File: Learning-Java/Arrays.java
/**
 * Arrays
 */
public class Arrays {

    public static void main(String[] args) {
        String[] cars = {"volvo", "bmw", "ford", "mazda"};
        int[] myNum = {10, 20, 30, 40};

        System.out.println(cars[1]);
        System.out.println(myNum[0]);
        System.out.println(cars.length);
        System.out.println(myNum.length);

        for (int i = 0; i < cars.length; i++) {
            System.out.println(cars[i]);
        }

        for (int i : myNum) {
            System.out.println(i);
        }
    }
}

File: learn-kotlin-w3c/arrays.kt
fun main(args: Array<String>) {
    val cars = arrayOf("volvo", "bmw", "ford", "mazda")
    cars[0] = "opel"
    println(cars[0])

    println(cars.size)

    if ("volvo" in cars) {
        println("exists")
    } else {
        println("it does not exists")
    }

    for (x in cars) {
        println(x)
    }

    val nums = arrayOf(1, 2, 3, 4, 5, 6)
    for (x in nums) {
        println(x)
    }

    // Ranges
    for (chars in 'a'..'x') {
        println(chars)
    }

    for (nums in 5..15) {
        println(nums)
    }


}
